gym/f110_gym/unittest/f110_env_test.py

```diff
@@ -105,3 +105,59 @@ def test_configure_method(self):
 
         base_env.close()
         extended_env.close()
+
+    def test_configure_action_space(self):
+        """
+        Try to change the upper bound of the action space, and check that the
+        action space is correctly updated.
+        """
+        base_env = self._make_env()
+        action_space_low = base_env.action_space.low
+        action_space_high = base_env.action_space.high
+
+        params = base_env.sim.params
+        new_v_max = 5.0
+        params["v_max"] = new_v_max
+
+        base_env.configure(config={"params": params})
```

Comment:
Does this test catch the issue in `configure()`?

Reply:
Good catch. 
The test was passing because of lines 118-120
```
params = base_env.sim.params
new_v_max = 5.0
params["v_max"] = new_v_max
```

So setting `params["v_max"] = new_v_max` was also updating the env params.

Fixed now.